command.py

```diff
@@ -66,15 +66,15 @@
 }
 
 multipart_command_types = {
-    'new_waypoint':             {'cmd':128, 'type':'ddffB'},
-    'insert_waypoint':          {'cmd':129, 'type':'ddffBBB'},
-    'set_returnHomeCoordinates':{'cmd':130, 'type':'ddf'},
-    'tare_IMU':                 {'cmd':131,  'type':'fff'},
-    'set_IMU':                  {'cmd':132,  'type':'fff'},
-    'set_KDValues':              {'cmd':133,  'type':'fffffff'},
-    'set_KPValues':              {'cmd':134,  'type':'fffffff'},
-    'set_KIValues':              {'cmd':135,  'type':'fffffff'},
-    'update_waypoint':          {'cmd':136, 'type':'ddffBB'},
+    'new_waypoint':             {'cmd':128, 'type':'ddffB'}, # lon, lat, alt, rad, type
+    'insert_waypoint':          {'cmd':129, 'type':'ddffxBB'}, # lon, lat, alt, rad, prev, next
```

Comment:
Ground station only gives it ddffBB, so the x will give padding to make sure it gets sent correctly. They commands that were changed are all parsed using the same struct on the picpilot, so adding these padding allows the WaypointWrapper to be used for all waypoint commands